atter-logic/src/commonMain/kotlin/com.attendance.checker/date/KDate.kt

```diff
@@ -0,0 +1,10 @@
+package date
+
+expect class KDate {
+
+    val isoDate: String
+
+    companion object {
+        fun now(): KDate
```

Comment:
My concern would be if such class design will be testable. Normally in Kotlin you pass `Clock` instance to have full control over time in you app.

Reply:
But do we want to test this class? It's `now()` method strongly depends on native implementation, so any tests should be performed directly in native scope (yea, doubled code, but this actually is platform specific issue).

Reply:
Not this class itself, but its usages. If you call `KDate.now()` anywhere in your project you won't be able to control its behavior. 
For example:
```
fun isAfter(otherTimestamp: Long)) : KDate{
  val date = KDate.now()
  return date.timestamp > otherTimestamp
}
```
I can't see any easy way to test it.